packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -131,7 +158,7 @@ export class NodeSDK {
   /**
    * Once the SDK has been configured, call this method to construct SDK components and register them with the OpenTelemetry API.
    */
-  public async start() {
+  public async start(): Promise<NodePluginManager> {
```

Comment:
What is the reason to return the plugin manager? Is there some reason you would need to have access to it after the loading process?

Reply:
just to have access to `NodePluginManager` methods like `stop()`. It was just an idea to handle the use of the plugin manager in this new package and I needed your feedback on it.

Reply:
We should focus on the metricProvider support inside plugins for this PR then maybe make another PR to expose more method on the pluginManager (if there is an use case)

Reply:
I updated it to not return anything. The only problem is that there is no reference to the plugin manager because it would be an unused variable and I couldn't use @ts-ignore.